.github/workflows/unit-tests.yaml

```diff
@@ -12,6 +12,25 @@ on:
     - reopened
     - synchronize
 jobs:
```

Comment:
Please rename `name: Tests` on the line `1` of this file to `name: Unit Tests`, otherwise the output I see in GitHub is confusing.
![image](https://user-images.githubusercontent.com/537715/212958936-553239e5-9469-431a-be4d-95f4bc2c7109.png)